configure.ac

```diff
@@ -419,7 +419,15 @@ if test x"${have_glib}" != xpkg; then
 fi
 
 if test x"${i_do_have_x}" = xyes; then
-   PKG_CHECK_MODULES([PANGO],[pango >= 1.10 pangoxft])
+   # pango depends on glib2. Required by GUI for {cairo or gtk/gtk2 or Mac CoreText}
+   # pangoxft is submodule of pango (used in gxcdraw) (there are other versions of Xft)
+   PKG_CHECK_MODULES([PANGO],[pango >= 1.10 pangoft2 pangoxft])
+   have_libpango=no
+   AC_CHECK_HEADER([pangoxft.h],
```

Comment:
Why do we need this check when we're already checking for pangoxft in PKG_CHECK_MODULES, this is failing travis